app/Http/Controllers/RegisterController.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Address;
+use App\Models\Token;
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Mail;
+use Illuminate\Support\Str;
+use App\Rules\CheckImg;
+use App\Rules\CheckName;
+use App\Rules\CheckBirthday;
+use App\Rules\CheckBlock;
+use App\Rules\CheckCity;
+use App\Rules\CheckPhoneNumber;
+use App\Rules\CheckKanaName;
+use App\Rules\checkPostalCode;
+use App\Rules\CheckPrefecture;
+use Exception;
+use Illuminate\Support\Facades\Storage;
+
+class RegisterController extends Controller
+{
+    public function index(Request $request)
+    {
+        return view('register');
+    }
+
+    public function sendMail(Request $request)
+    {
+        $email = $request->email;
+        $this->validate($request,[
+            'email' => 'email:filter,d|unique:users,email'
+        ]);
+
+        $tokentable = new Token();
```

Comment:
変数とモデル名はできるだけ合わせてほしい
```suggestion
        $token = new Token();
```

Reply:
変数名とモデル名を統一致しました。
ご確認よろしくお願い致します。
app/Http/Controllers/TokenController.php
app/Http/Controllers/RegisterController.php